srcpkgs/black/template

```diff
@@ -1,19 +1,38 @@
 # Template file for 'black'
 pkgname=black
-version=20.8b1
+version=21.8b0
 revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools_scm"
-depends="python3-appdirs python3-click python3-mypy_extensions
- python3-pathspec python3-toml python3-regex python3-typed-ast
+hostmakedepends="python3-setuptools python3-setuptools_scm"
+depends="python3-click python3-platformdirs python3-tomli
+ python3-mypy_extensions python3-pathspec python3-regex
  python3-typing_extensions"
+checkdepends="python3-typed-ast python3-aiohttp python3-aiohttp-cors
+ python3-pytest python3-parameterized python3-ipython python3-tokenize-rt
+ $depends"
 short_desc="Uncompromising Python code formatter"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
-homepage="https://github.com/psf/black"
+homepage="https://pypi.org/project/black"
```

Comment:
The pypi page links github as the homepage and in the past I saw some projects not publishing newer versions on pypi, so I would not change the homepage.